tutorials/creating_sequences.ipynb

```diff
@@ -4,543 +4,308 @@
-   },
-   "source": [
-    "By looking at the current schedule, we can see that `'ch0'` was delayed from `ti=220` to `tf=1100`, and only then was the `complex_pulse` added. The reason for this is simple: it had to wait for the pulse on `ch1`, also targeted to qubit `4`, to finish before it could apply its own. It behaved this way because, in `Sequence.add` there is a default argument `protocol='min-delay`."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Another protocol for pulse addition is `wait-for-all`, which makes the new pulse wait even if there is no conflict. Let's remove the conflict and add two `simple_pulse`s to `ch1` with the different protocols to see how the compare."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "seq.target(\"q0\", \"ch1\")\n",
-    "seq.add(simple_pulse, \"ch1\", protocol=\"min-delay\")\n",
-    "seq.add(simple_pulse, \"ch1\", protocol=\"wait-for-all\")\n",
+    "Since $H=H^D$, the final state after a pulse of duration $\\Delta t$, constant amplitude $\\Omega$, detuning $\\delta=0$ and phase $\\phi=0$ is:\n",
     "\n",
-    "print(\"Current Schedule:\")\n",
-    "print(seq)\n",
-    "seq.draw()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Because we removed the conflict by changing the target of `ch1` to qubit `0`, we see that the first pulse was added without delay from `ti=1100` to `tf=1300` (i.e. while `complex_pulse` is still running in `ch0`). However, once we changed to `protocol='wait-for-all'`, there is now a delay (from `ti=1300` to `tf=2100`) that idles `ch1` until `ch0`is finished with its pulse, even though there was no conflict."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "The third protocol is called `'no-delay'` which, as the name implies, never delays the channel where the pulse is being added, even if that means introducing a conflict."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "seq.target(\"q0\", \"ch0\")\n",
-    "seq.add(complex_pulse, \"ch0\", protocol=\"no-delay\")\n",
+    "$$\n",
+    "\\begin{align}\n",
+    "\\left|\\Psi_f\\right> &= \\exp\\left(-i \\frac{\\Omega}{2} \\Delta t (|g\\rangle\\langle r| + |r\\rangle\\langle g|)\\right) \\left|g\\right> \\\\\n",
+    "&= \\left(I \\cos\\left(\\frac{\\Omega}{2} \\Delta t\\right) - i \\sin\\left(\\frac{\\Omega}{2} \\Delta t\\right)(|g\\rangle\\langle r| + |r\\rangle\\langle g|)\\right)\\left|g\\right>\n",
+    "\\end{align}\n",
+    "$$\n",
+    "\n",
+    "From which we get that for $\\frac{\\Omega}{2} \\Delta t = \\frac{\\pi}{2}$, the final state should be equal to $\\left|r\\right>$ (up to a phase, that is irrelevant when measuring with cold atoms since we measure $\\left|\\left<\\Psi_f\\right|\\left|r\\right>\\right|^2$, see below).\n",
```

Comment:
I think this is too much for this tutorial. We can perhaps write it in another page and refer to it here, It make sense as a collapsing section in the Programming page